src/components/PageDefinitions.js

```diff
@@ -249,6 +255,35 @@ export class PageDefinitions extends AbstractPageDefinitions {
     this.copyToClipboard(url, 'URL copied to clipboard')
   }
 
+  doSaveAsGist(fileName, fileContent) {
+    const { token } = this.props
+
+    createGist(token, {
+      files: {
+        [fileName]: {
+          content: fileContent
+        }
+      }
+    })
+      .then(res => {
+        return this.props.dispatch(
+          uiNotificationNew({
+            type: 'info',
+            message: (
+              <div>
+                Gist File has been created is available{' '}
```

Comment:
This message doesn't make much sense. can you review it?

Reply:
I've changed the label as you suggested, please check it out